services/inputhost/pathCache.go

```diff
@@ -508,3 +508,16 @@ func (pathCache *inPathCache) getState() *admin.DestinationState {
 
 	return destState
 }
+
+func (pathCache *inPathCache) drainExtent(extUUID string) error {
+	pathCache.RLock()
+	defer pathCache.RUnlock()
+
+	extCache, ok := pathCache.extentCache[extentUUID(extUUID)]
+	if !ok {
+		return &cherami.EntityNotExistsError{}
+	}
+
+	go extCache.connection.drain()
```

Comment:
No.. I didn't have it planned that way, the timeouts are enforced on the drain layers and the controller will get an ExtentUnreachableNotification() call or a node failed event if it crashes.